src/internal/DotStarEsp32DmaSpiMethod.h

```diff
@@ -191,7 +208,7 @@ template<typename T_SPISPEED, typename T_SPIBUS> class DotStarEsp32DmaSpiMethod
 
     void Update(bool)
     {
-        while(!IsReadyToUpdate());
+        while(!IsReadyToUpdate()) portYIELD();
```

Comment:
minor:  Style for this code base is always use open and close curly braces and one statement per line.